atd-vzv/src/views/summary/CrashesByMode.js

```diff
@@ -174,6 +174,18 @@ const CrashesByMode = () => {
     datasets: !!chartData && createTypeDatasets(),
   };
 
+  // Get an array of annual totals for the selected crash type
+  const getYearTotalsArray = () => {
+    const yearTotalsArray = yearsArray().map((year, index) => {
+      let currentYearTotal = 0;
+      data.datasets.forEach((mode) => {
+        currentYearTotal += mode.data[index];
+      });
+      return currentYearTotal;
```

Comment:
[reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) could be fun alternative here. Not a change request as this is great too!

I once had a code block like this with declaration + work + return described to me as a "code sandwich". I've never been able to unsee it. 🥪

Reply:
## 🥪 🤯❗

What a great name for that. I think it just got burnt into my mind too,

Reply:
Haha glad you enjoyed - felt like sharing that funny story. 😄 Yep, reduce can be hard to follow sometimes too - so many ways to do things! I think the code here is great. 🚀

Reply:
ahh i never responded to this -- yeah code sandwich is accurate lol. theres often that choice between picking an elegant/less lines of code way vs something that is easier to read/follow but more wordy... sometimes i like the easy to read way but i need to check out `reduce` some time :p